generators/client/templates/vue/src/main/webapp/app/shared/jhi-item-count.component.ts.ejs

```diff
@@ -0,0 +1,24 @@
+import Component from 'vue-class-component';
```

Comment:
check the indent of this file : it should be with 2 spaces, not 4

Reply:
You're right.